pshtt/pshtt.py

```diff
@@ -127,6 +131,12 @@ def result_for(domain):
 
 
 def ping(url, allow_redirects=False, verify=True):
+    # If there is a custom CA file and we want to verify
+    # use that instead when pinging with requests
+
+    if CA_FILE and verify:
+        verify = CA_FILE
```

Comment:
Is this to trigger the right behavior in `requests`, if you set `verify` to be a path to a `.pem` file rather than a boolean? If so, a comment to that effect would be helpful, as it took me a bit to realize the reason for this assignment.